_includes/samples/PriceFeeds/ArbitrumPriceConsumer.sol

```diff
@@ -56,7 +56,7 @@ contract ArbitrumPriceConsumer {
     ) = sequencerUptimeFeed.latestRoundData();
 
         // Answer == 0: Sequencer is up
-        // Snswer == 1: Sequencer is down
+        // Answer == 1: Sequencer is down
         if (answer == 0 && (block.timestamp - updatedAt) < 3600) {
```

Comment:
Let's use `startedAt` here to be consistent to our `Optimism` example

Reply:
Done.